index.html

```diff
@@ -48,32 +48,199 @@ <h2>I'm a Software Developer</h2>
+                    <p class="flex-items item2 projectitem2">HTML</p>
+                    <p class="flex-items item3 projectitem3">BootStrap</p>
+                </div>
+                <button class="work-btn projbtn">See Project</button>
+            </div>
+            <div id="proj4">
+                <h2 class="work-subheading projecttitle">Profesional Art Printing Data</h2>
+                <p class="work-specs projectdetails">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.</p>
+                <div class="for-flex projectflex2">
+                    <p class="flex-items item1 projectitem1">CSS</p>
+                    <p class="flex-items item2 projectitem2">HTML</p>
+                    <p class="flex-items item3 projectitem3">BootStrap</p>
+                </div>
+                <button class="work-btn projbtn">See Project</button>
+            </div>
+        </section>
+        <section class="projthirdpage">
+            <div id="proj5">
+                <h2 class="work-subheading projecttitle">Profesional Art Printing Data</h2>
+                <p class="work-specs projectdetails">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.</p>
+                <div class="for-flex">
+                    <p class="flex-items item1 projectitem1">CSS</p>
+                    <p class="flex-items item2 projectitem2">HTML</p>
+                    <p class="flex-items item3 projectitem3">BootStrap</p>
+                </div>
+                <button class="work-btn projbtn">See Project</button>
+            </div>
+            <div id="proj6">
+                <h2 class="work-subheading projecttitle">Profesional Art Printing Data</h2>
+                <p class="work-specs projectdetails">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.</p>
+                <div class="for-flex projectflex2">
+                    <p class="flex-items item1 projectitem1">CSS</p>
+                    <p class="flex-items item2 projectitem2">HTML</p>
+                    <p class="flex-items item3 projectitem3">BootStrap</p>
+                </div>
+                <button class="work-btn projbtn">See Project</button>
+            </div>
+        </section>
+        <section id="about-me">
+            <h2 id="for-me">About me</h2>
+            <p>
+                I can help you to build a product, feature or website. Look through
+                some of my work and experience! If you like what you see and have a
+                project you need coded, don't hesitate to contact me.
+            </p>
+            <button class="work-btn resume-btn">Get My Resume</button>
+            <div><img id="aboutme-img" src="/img/illustration-about-me-mobile.png" alt=""></div>
```

Comment:
Since this illustration is only for design, consider using it as a background instead of as an `img` tag to not confuse users on screen readers